<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>L2tpWidget</class>
 <widget class="QWidget" name="L2tpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>388</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Gateway:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="gateway"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>User name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="username"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <widget class="KLineEdit" name="password">
       <property name="passwordMode">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboUserPasswordType">
       <item>
        <property name="text">
         <string>Store</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Always Ask</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Not Required</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="cbShowPassword">
     <property name="text">
      <string>Show password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>NT Domain:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="domain"/>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>216</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnIPSecSettings">
       <property name="text">
        <string>IPsec Settings...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPPPSettings">
       <property name="text">
        <string>PPP Settings...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>gateway</tabstop>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>cboUserPasswordType</tabstop>
  <tabstop>cbShowPassword</tabstop>
  <tabstop>domain</tabstop>
  <tabstop>btnIPSecSettings</tabstop>
  <tabstop>btnPPPSettings</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
